spec/features/idv/doc_auth/upload_step_spec.rb

```diff
@@ -21,11 +24,19 @@
     it 'proceeds to send link via email page when user chooses to upload from computer' do
       click_on t('doc_auth.info.upload_computer_link')
       expect(page).to have_current_path(idv_doc_auth_email_sent_step)
+      expect(fake_analytics).to have_logged_event(
+        'IdV: ' + "#{Analytics::DOC_AUTH} upload submitted".downcase,
```

Comment:
seems weird to me to mix `+` and `#{}` string interpolation on the same line... I would do this?
```suggestion
        "IdV: #{Analytics::DOC_AUTH.downcase} upload submitted",
```

Reply:
Agree! These were copied from [a similar test case](https://github.com/18F/identity-idp/blob/4592736065e659a4af2159c62c204476b652d489/spec/features/idv/doc_auth/agreement_step_spec.rb#L82), but in hindsight I much prefer your alternative.